com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/helpers/ClientHelper.java

```diff
@@ -24,8 +23,8 @@ public ClientHelper(CloudControllerClient client) {
     }
 
     public void deleteRoute(String uri) {
-        Pair<String, String> hostAndDomain = UriUtil.getHostAndDomain(uri);
-        client.deleteRoute(hostAndDomain._1, hostAndDomain._2);
+        ApplicationURI route = new ApplicationURI(uri);
```

Comment:
has this been tested to work like the UriUtil.gethostAndDomain(uri) in all cases?